modules/installation-gcp-permissions.adoc

```diff
@@ -30,31 +31,39 @@ When you attach the `Owner` role to the service account that you create, you gra
 .Required roles for creating network resources during installation
 * DNS Administrator
 
-.Required roles for using passthrough credentials mode
+.Required roles for using the Cloud Credential Operator in passthrough mode
 * Compute Load Balancer Admin
-* IAM Role Viewer
```

Comment:
**Passthrough mode**

Removed "IAM Role Viewer" per Jira comment; is "Compute Load Balancer Admin" still valid?

Reply:
I would assume so.